src/pathfinding_service/model/token_network.py

```diff
@@ -148,17 +148,18 @@ def _get_address_metadata(
         # Check node reachabilities
         metadata: Dict[str, Dict[str, Union[str, PeerCapabilities]]] = {}
         for node in self.nodes:
-            node_user_ids = self.reachability_state.get_userids_for_address(node)
+            node_users = self.reachability_state.get_users_from_address(node)
```

Comment:
`get_users_from_address()` will perform a search on the server to find users that belong to an address. We should only rely on the presence updates received by the server reactively and not perform a search on the server. Only take what we have in the state.